### ISSUE TITLE
Disappearing ActionBar after finishing ActionMode

### ISSUE THREAD

**Issue by KamilLelonek** [JakeWharton/ActionBarSherlock#1014]
I've implemented `ActionMode` with `SherlockActionBar` and everything works fine, but only on devices post `HONEYCOMB`.

On older `APIs` I have the following problem:

Everything is fine when I start my app:

![enter image description here](http://i.stack.imgur.com/mZ5Ff.png)

All goes well even when I start action mode and select a couple of items:

![enter image description here](http://i.stack.imgur.com/46zve.png)

But when I exit (finish) action mode `ActionBarSherlock` crashes. I mean that it's no loger visible, only when I start or finish action mode, it blinks very shortly that I can see icons:
![enter image description here](http://i.stack.imgur.com/EzssT.png)

So only updates show ActionBar for a short moment (it blinks), but after that it remains "hidden" (covered) again.

Has anyone similar problem? Is there any sollution for that?

**Comment by KamilLelonek** [comment]
![Preview](http://i.imgflip.com/30u4g.gif)

**Comment by bradmcmanus** [comment]
I am seeing the same exact behavior. Are you using ABS 4.3? I am investigating it now. I'm going to try updating to latest, too.

**Comment by KamilLelonek** [comment]
Yes, I've tested it with both 4.2 and 4.3.

**Comment by rishabhmhjn** [comment]
I had the same problem in an Android version 4.3 device

**Comment by bradmcmanus** [comment]
Ah, strange. I only see the unwanted behavior on a device running Gingerbread. I suspected it was a problem with the ported ActionMode and other ActionBar related classes necessary for 2.3 devices.

**Comment by rishabhmhjn** [comment]
My problem is when I scroll through a list in the activity, the actionbar disappears and come backs upon touching at its position. I do not use action mode though.

**Comment by KamilLelonek** [comment]
Are u  using some custom list view or something or everything goes the "normal way"?

**Comment by bradmcmanus** [comment]
Seems to be a problem either way. In one case the ActionBar is using a customView, in another case the ActionBar just has an icon and title set on it.

**Comment by bradmcmanus** [comment]
FYI, we tracked it down to being an issue with NOA. We removed the ABS included NOA in favor of the NOA jar, and this was the source of the contextual action bar issue.

**Comment by KamilLelonek** [comment]
What does NOA stand for? Is the problem solved?

**Comment by bradmcmanus** [comment]
Ah, NineOldAndroids. ABS includes a stripped down version of NineOldAndroids in its source but there is also a standalone NineOldAndroids library with more features. Our problem was resolved by making sure to use the com.actionbarsherlock.internal.nineoldandroids (as happens by default, unless you modify the ABS source).

**Comment by KamilLelonek** [comment]
I use modified ABS without NOA package, which is provided from external library project, because previous configuration (NOA package and NOA library which is required by project) caused conflicts.

**Comment by jenzz** [comment]
Yep. I can confirm using the **com.actionbarsherlock.internal.nineoldandroids** methods instead of an external .jar fixes this issue.
